stl/inc/random

```diff
@@ -490,6 +490,16 @@ _NODISCARD _Uint _Next_linear_congruential_value(_Uint _Prev) noexcept {
         // numeric_limits<result_type>::max() plus 1." That is: Just do the multiply
         // and let normal unsigned modulo take care of it
         return static_cast<_Uint>(static_cast<_Uint>(_Ax * _Prev) + _Cx);
+    } else if constexpr (_Cx == 0 && _Mx == 2147483647) {
+        // minstd_rand
+        constexpr unsigned long long _All = static_cast<unsigned long long>(_Ax);

```

Comment:
Would be interesting to have benchmark results on 32-bit x86

Reply:
x86 benchmark:
BEFORE:
----------------------------------------------------------------
Benchmark                      Time             CPU   Iterations
----------------------------------------------------------------
BM_lcg                      14.9 ns         15.1 ns     49777778
AFTER:
BM_lcg                      4.36 ns         4.39 ns    160000000
3x speedup

for apples-to-apples comparison
x64 benchmark on the same box:
BEFORE:
BM_lcg                      7.55 ns         7.50 ns     89600000
AFTER:
BM_lcg                      3.55 ns         3.57 ns    179200000
2x speedup